maxim/logger/together/client.py

```
@@ -0,0 +1,235 @@
+            
+            # After stream is complete, log the accumulated result
+            if generation is not None:
+                # Create a complete response object
+                response_to_parse = TogetherUtils.parse_chunks_to_response(accumulated_content, final_usage)
+                generation.result(TogetherUtils.parse_completion(response_to_parse))
+            
+            if is_local_trace and trace is not None:
+                trace.set_output(accumulated_content)
+                trace.end()
+                
+        except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)